src/std/os/signal-handler.ss

```diff
@@ -60,31 +61,25 @@ package: std/os
         (log-error "Error handling signals" e)
         (raise e))))
 
-   (def (signal-handler-dispatch handler)
-     (try
-      (handler)
-      (catch (e)
-        (log-error "error dispatching signal handler" e))))
-
    (def (signal-handler-add! sh signo thunk)
      (with ((signal-handler tab sfd sigset mx) sh)
        (with-lock mx
-         (lambda ()
-           (when (fxzero? (sigismember sigset signo))
-             (sigaddset sigset signo)
-             (block-signal! signo)
-             (signalfd-reset! sfd sigset))
-           (vector-set! tab signo thunk)))))
+	 (lambda ()
```

Comment:
this should be indented at level 1 -- `M-x eval (gerbil-put-indent '(with-lock) 1)`